Gemfile

```diff
@@ -8,10 +8,15 @@ group :development, :test do
   gem 'pry' unless ENV['CI']
   gem 'pry-byebug' unless ENV['CI']
   gem "simplecov", require: false
+  # Override the coffee-rails version in engine_cart for rails 4.2
+  case ENV['RAILS_VERSION']
+  when /^4.2/
+    gem 'coffee-rails', '~> 4.2.0'
```

Comment:
Good question - I have not tested what happens when the rails version hits this code.  If hyrax is no longer compatible with rails 4.x, then I should just remove the last commit and leave that stanza as engine-cart creates it (i.e. discard the override entirely).